### ISSUE TITLE
Integrate reporting engine vignettes into the online documentation

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/docs#113]
Should we integrate RE-vignettes as we did for the [R-Toolbox](https://docs.open-systems-pharmacology.org/working-with-r/r-introduction)?

If yes: the following problem must be solved:
Currently I convert html-vignettes via pandoc like this:
`pandoc introduction-re-instances.html -o introduction-re-instances.md --extract-media=Images -t gfm -f html-native_spans`

Works fine with the only exception: **example report outputs style** (background etc.) is not properly converted.
So e.g. in a vignette it looks like this:

![grafik](https://user-images.githubusercontent.com/25061876/98142500-294aea00-1ec8-11eb-971e-27cf6313c192.png)

And in the [converted markdown](https://github.com/Yuri05/DisTest01/blob/main/introduction-re-instances.md):

![grafik](https://user-images.githubusercontent.com/25061876/98142613-4e3f5d00-1ec8-11eb-8282-5ad7a51a69e6.png)

**Comment by Yuri05** [comment]
@msevestre @pchelle @abdullahhamadeh Any thoughts on this?

**Comment by msevestre** [comment]
If it can be automated yes, otherwise no

**Comment by Yuri05** [comment]
Should we try to include **htmls** of vignettes instead of markdowns?
There are some plugins allowing this (e.g. https://github.com/Bandwidth/gitbook-plugin-include-html ) - no idea how good it works though..

**Comment by Yuri05** [comment]
Yep, no need to integrate the vignettes anymore, the current pkgdown solution is much better :)
